pkg/orchestration/wiring/k8scompute/plugin.go

```diff
@@ -539,11 +537,22 @@ func buildAntiAffinity(labelMap map[string]string) *core_v1.PodAntiAffinity {
 	}
 }
 
-func buildPodDisruptionBudgetSpec(labelMap map[string]string) policy_v1.PodDisruptionBudgetSpec {
+func buildPodDisruptionBudgetSpec(labelMap map[string]string, minReplicas int32) policy_v1.PodDisruptionBudgetSpec {
+	// Make sure we can drain nodes if the minReplicas is < 3
+	var pdbPercentage string
+	switch minReplicas {
+	case 0, 1:
+		pdbPercentage = "0%"
+	case 2:
+		pdbPercentage = "50%"
+	default:
+		pdbPercentage = "66%"
```

Comment:
Is it okay for more than half the Pods to be disrupted? Should we maybe have 33% for replicas >= 3 ? Or maybe even make the `case 2` the default, for 50% with replicas >= 2 ?
If these were EC2s, we'd be this conservative, but I don't know if we can be more disruptive with Pods, maybe?

Reply:
This is a poorly named variable.  I made this mistake myself.  We are configuring "minimum available" here, eg. we want at least 2/3rds of the nodes to be available when > 3 nodes.

I'll rename the variable to make this more clear.